src/mame/layout/irrmaze.lay

```diff
@@ -21,6 +21,11 @@ copyright-holders:Vas Crabb
 		<rect><color red="0.5" green="0" blue="0" /></rect>
 	</element>
 
+	<element name="colorlamp">
+		<rect state="0"><color red="0.4" green="0" blue="0" /></rect>
+		<rect state="1"><color red="1" green="0.1" blue="0.25" /></rect>
+	</element>
```

Comment:
These are incandescent globes with red glass behind a grey marquee element.  They shouldn't be pink when lit.